### PR TITLE
Duplicate review

### PR DESCRIPTION
**Pull request by jacobtoppm**
Adds duplicate review stage in js
Adds function to find most likely duplicate for drive data using a specified mapping and field weighting
Adds tests
Adds mixin that can be added to a custom image model to add additional fields for identifying duplicates

### CODE REVIEW

File: wagtail_image_import/.babelrc [hunk 1]
```diff
@@ -1,3 +1,13 @@
 {
-    "presets": ["@babel/preset-env", "@babel/preset-react"]
-}
\ No newline at end of file
+    "presets": [
+      [
+        "@babel/preset-env",
+        {
+          "targets": {
+            "node": "10"
```
Comment (KalobTaulien) [inline wagtail_image_import/.babelrc:7]: Is it worth creating an `.nvmrc` file to specify the node version as well?

File: wagtail_image_import/models.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+from wagtail.images import get_image_model_string
+
+
+class DriveIDMapping(models.Model):
+    """
+    Represents the Drive ID of an image which has previously been imported from Google Drive, used for identifying duplicates/reimports
+    """
+
+    image = models.OneToOneField(get_image_model_string(), on_delete=models.CASCADE)
+    drive_id = models.CharField(max_length=100)
```
Comment (KalobTaulien) [inline wagtail_image_import/models.py:16]: Just to confirm, max_length of a drive id will always be <= 100 characters right? As in, Google doesn't state they could be up to 128 characters?

File: wagtail_image_import/views.py [hunk 3]
```diff
@@ -2,18 +2,62 @@
 
 from django.conf import settings
 from django.core.exceptions import PermissionDenied
+from django.http import HttpResponse, JsonResponse
```
Comment (KalobTaulien) [inline wagtail_image_import/views.py:5]: I don't believe HttpResponse is being used in this file

File: tests/tests/test_duplicates.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        duplicate = get_most_likely_duplicate(
+            {"md5Checksum": "93d85f960bcffa9c1b1d55296db40ad0"}, *hash_config
+        )
+        self.assertEqual(duplicate, self.wagtail_1_image)
+        duplicate_2 = get_most_likely_duplicate(
+            {"md5Checksum": "93d85f960bcffa345b1d55296db40ad0"}, *hash_config
+        )
+        self.assertEqual(duplicate_2, None)
+
+    def test_find_exif_datetime_duplicates(self):
```
[the remark this answers is not in the document]
Comment (jacobtoppm) [reply tests/tests/test_duplicates.py:80]: Thanks! Fixed now

File: wagtail_image_import/.babelrc (hunk 1 above)
Comment (jacobtoppm) [reply wagtail_image_import/.babelrc]: Good point, I've added in latest commit - thanks!

File: wagtail_image_import/views.py (hunk 3 above)
Comment (jacobtoppm) [reply wagtail_image_import/views.py]: Removed now, thanks!

**Pull request merged**